src/main/java/com/bulkbuy/enterprise/dto/Order.java

```diff
@@ -15,8 +15,13 @@
     private Date estimatedArrival;
     private String carrier;
     private String orderStatus;
-    private int itemID;
+    private int itemId;
```

Comment:
Updated the name for itemID to itemId to follow standard naming conventions guidelines for variables.

Reply:
For acronyms, the naming conventions are not always clear.  Either approach are often accepted.